cmd/namespace-divergence/README.md

```diff
@@ -0,0 +1,17 @@
+Namespace Divergence
+
+Namespace Divergence is a tool that cross-references the namespaces in your Kubernetes cluster with the namespaces directory in the GitHub repository `ministryofjustice/cloud-platform-environments`. If a divergence is found, it can send a slack message or just print to screen. This way, you can be sure that accedental namespaces don't exist and ensure your data is always up-to-date, without spending hours doing manual checks.
+
+You can run the namespace divergence command if you have go by running:
+
+```bash
+go run divergence.go -p <personal_access_token> -kubeconfig <path_to_kubeconfig> -slackApi <OAUTH_token> -channelId <slack_channel_id_integer>
```

Comment:
Is it worth specifying `<slack_channel_id>` i.e. removing `_integer`, as Slack channel IDs are alphanumeric?

Reply:
I didn't know this, but yes, I'll get it changed :). Thanks